### PR TITLE
new bismark-opkg-keys first commit

### PR DESCRIPTION
**Pull request by ggmartins**
Once deploy-release.sh is executed, the serverCA.pem has to replace the default serverCA.pem inside this package. Then the package must be rebuilt/deployed.

### CODE REVIEW

File: utils/bismark-opkg-keys/Makefile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	$(INSTALL_DIR) $(1)/etc/ssl
+	$(INSTALL_DATA) ./files/etc/ssl/serverCA.pem $(1)/etc/ssl/serverCA.pem
+endef
+
+define Package/bismark-opkg-keys/postinst
+#!/bin/sh                                                                                                                          
+
+if [ -z "$${IPKG_INSTROOT}" ]; then                                                                                                                                   
+    if  ! grep -Fs "signature" /etc/opkg.conf >/dev/null ; then
+        sed -i '$$aoption check_signature 1'  /etc/opkg.conf
```
Comment (sburnett) [inline utils/bismark-opkg-keys/Makefile:42]: Why not just:

```
echo "option check_signature 1" >> /etc/opkg.conf
...
```

I don't understand sed well enough to know what the `$$a` at the beginning of the pattern means.

Comment (ggmartins) [inline utils/bismark-opkg-keys/Makefile:42]: $a it's an append cmd.. the $$ is to make the openwrt packaging system
ignore any instruction.

Comment (sburnett) [inline utils/bismark-opkg-keys/Makefile:42]: On Fri, Oct 25, 2013 at 12:25:54PM -0700, ggmartins wrote:

> > +endef
> > +
> > +define Build/Compile
> > +endef
> > +
> > +define Package/bismark-opkg-keys/install
> > -   $(INSTALL_DIR) $(1)/etc/ssl
> > -   $(INSTALL_DATA) ./files/etc/ssl/serverCA.pem $(1)/etc/ssl/serverCA.pem
> >   +endef
> >   +
> >   +define Package/bismark-opkg-keys/postinst
> >   +#!/bin/sh  
> >   +
> >   +if [ -z "$${IPKG_INSTROOT}" ]; then  
> > -    if  ! grep -Fs "signature" /etc/opkg.conf >/dev/null ; then
> > -        sed -i '$$aoption check_signature 1'  /etc/opkg.conf
> 
> $a it's an append cmd.. the $$ is to make the openwrt packaging system
> ignore any instruction.

Cool, didn't know aobut $a in sed. Thanks for the tip.

**Comment by sburnett** [comment]
Please increment the release number on `opkg-bismark`.

**Pull request merged**